### ISSUE TITLE
Frash : Ipad 2 under ios 4.3.3 error : abort () food.c:43

### ISSUE THREAD

**Issue by Therabbit** [comex/frash#29]
Hello, I have jaibreak my Ipad 2 under ios 4.3.3 and I have installed Frash ( http://repo.benm.at) And I have an error when I click  on the "frash" on the web site (ex: canal+) I have an error : abort () food.c:43. I have uninstall and install frash and reboot, but That walks(works) always not. Why?

**Comment by ifonefox** [comment]
This is an issue occurring on all iOS devices ever since 4.2.1. See https://github.com/comex/frash/issues/28 for more info.

**Comment by Therabbit** [comment]
ok thank you

**Comment by Therabbit** [comment]
And if I change the ios of my ipad. That will walk(work)?

**Comment by ifonefox** [comment]
If you are able to downgrade it to 4.1, then yes. I don't think 4.1 ever came to the iPad 2, though.

**Comment by Therabbit** [comment]
Then this will walk or not because I downloaded many applications

**Comment by Therabbit** [comment]
Which other firmware walk? Because I didn't find firmware 4.1 on google

**Comment by ifonefox** [comment]
It has to be 4.1 or older

**Comment by Therabbit** [comment]
And you know where I can find him(it)? WiFi Ipad 2 32g

**Comment by Therabbit** [comment]
I didn't find Firmware 4.1 for ipad 2 or older on google

**Comment by ifonefox** [comment]
I don't think it ever came out for the iPad 2. For flash on the ipad, you may want to check out the application "iSwifter". It is a web browser that allows you to use flash. http://itunes.apple.com/us/app/iswifter/id388857173?mt=8

**Comment by Therabbit** [comment]
Ok

**Comment by Therabbit** [comment]
I have another question, there are risks of virus when I download musics via safari? If yes that to make?
